### DISCUSSION TITLE
Moved code-server to another location and it broke

### DISCUSSION THREAD

**Discussion by dedfft** [coder/code-server#4150] [Q&A]
My initial server was in frankfurt on digital ocean. But i made a snapshot and moved it to France. 
Now when i try to access via ip it sends me to https://github.com/bpmct/coder-cloud-redirect-server/blob/master/README.md#troubleshooting

i checked logs via journalctl -u code-server@coder
And empty.
Sudo reboot now allowed to open something. But now it gives me error `Error connecting to your code-server instance

Ensure code-server is running and was properly started with the --link command line flag. 

If you would like to talk to a developer, drop a message in our slack channel. `

the auth method is Tls via github

**Comment by lg-nineoneninethreenine** [comment]
@dedfft it was clearly stated in the README that your redirect works but is unable to read the status. Did you try the commands that was listed there?

**Reply by lg-nineoneninethreenine** [reply]
Try running code-server only, no link tags or anything, just `code-server` and try again.

**Comment by dedfft** [comment]
Yes, tried all commands and they didn’t help.
Also code-server command 
Returns screenshot. But still same error page
![BCF9A332-C852-489C-B144-58CB4017876E](https://user-images.githubusercontent.com/43168887/133051714-9014d2e7-d447-4f3a-bad4-b2b1cb513a5f.jpeg)

`Error connecting to your code-server instance

Ensure code-server is running and was properly started with the --link command line flag. 

If you would like to talk to a developer, drop a message in our slack channel. `

**Reply by lg-nineoneninethreenine** [reply]
I think you will have to drop a Slack message. If you can't, I will ask for help there.

**Reply by dedfft** [reply]
Thank you. How can i join slack? It says that i am not in your organization

**Reply by lg-nineoneninethreenine** [reply]
You must provide your email and Slack will send a verification code to that email. After that, set your name and send a message with this discussion link on the code-server channel. If you can't, I can do it since I formed a Slack account.

**Comment by dedfft** [comment]
Tried fresh clean install without any images. It worked yesterday perfectly. But same error with the new fresh installation. 
Followed this https://github.com/cdr/deploy-code-server/blob/main/guides/digitalocean.md
<img width="886" alt="Снимок экрана 2021-09-13 в 11 49 42" src="https://user-images.githubusercontent.com/43168887/133053789-4427cb66-6014-47a5-ba91-4d120d57ffeb.png">


Update:
Tried 2 more times with different droplets. Exactly same as yesterday. I am unable to launch it(. Please, any ideas. 

update2:
I am going crazy. I created a new code-server on different cloud provider, launched and it worked. But it works only while ssh is connected. 

update3:
the only working codeserver on the first droplet(yesterday) somehow configured itself to use github only auth without ssh connection present to this server.

**Reply by bpmct** [reply]
Hi @dedfft. Would you be able to [join our Slack](https://cdr.co/join-community). Would love to debug with you. We have a new version of Link coming very soon with improved networking and servers, so we will be sure you are on that.

**Reply by lg-nineoneninethreenine** [reply]
Just a suggestion: Try the same thing on a new server, since the old one has issues, how can a new server have it? Try making a new server and try again.
